BasiliskII/src/rom_patches.cpp

```diff
@@ -1008,6 +1008,179 @@ static bool patch_rom_classic(void)
 	*wp++ = htons(M68K_EMUL_OP_IRQ);
 	*wp++ = htons(0x4a80);		// tst.l	d0
 	*wp = htons(0x67f4);		// beq		0x402be2
+
+
+	// Patch guest screen for an arbitary resolution
+	// We are going to steal the ROM space that runs RAM test
+	uint32 patchCodeBaseMac; 	// a stateful variable to keep track of the stolen location of the patched code in guest memory.
```

Comment:
Nit: Why not declare this variable on the line it's first assigned (line 1026)? Also, I find the comment would be more readable to be above the line.

Reply:
I want to single out them to add comments.

I moved all comment (none assembly) above the line